README.md

```diff
@@ -1,23 +1,129 @@
+  </a>
+    
+</p>
+
+<p align="center">
+  <a href="#key-features">Key Features</a> •
+  <a href="https://www.youtube.com/watch?v=d6ds5Lch3GA">Presentation video</a> •
+  <a href="#how-to-use">How To Use</a> •
+  <a href="#screenshots">Screenshots</a> •
+  <a href="#the-team">The team</a>
+</p>
 
-## Description
 The purpose of this app is to offer emergency help by allowing willing users to register and enter their skills (e.g. have taken courses on first aid) and the first aid medicine they usually walk around with (eg. EpiPens for people with severe allergies, asthma inhalers for people with asthma, antihistamines for allergies…).
 
 When a user suddenly needs their medication (e.g. if they had an anaphylactic shock), after calling an ambulance, them (or the people they are with) can go on the app and ask for a certain medication or for someone with a particular first aid skill, the app will then notify any matching users in a certain radius so that they can come and help.
 
+It was created as part of the Software Development Project [CS-306](https://edu.epfl.ch/coursebook/en/software-development-project-CS-306-1)
+class at EPFL.
+
+## Key Features
+
+Once the app is downloaded the users can sign in and fill up their information:
+- sign in with Google or anonymously. 
+- Fill up their medical information, containing useful information in case of emergency like blood type, allergy or current Treatment.
+- Add  an emergency contact to call in case of emergency.
+- Add there medical skills and the medicine they carry with them.
+
+That's it they are ready to save and be saved.
+
+In case of an emergency:
+- tap the H3LP button and select what help he/she needs
+- can decide to call the emergency service or her/his emergency contact
+- chat with the helpers that are comming for her/him  to give them more information
+
+
+## Other Features
+
+H3LP is a lot more than that and contains many features to keep you safe:
+- A map to locate nearby hospitals, pharmacies and defibrillators
+- Many tutorials to learn first aid tips with videos
+- A simple tool that gives you the best tempo to perform CPR
+- A forum where users can ask medical professionals for tips and information by topic
 
-## The Team
+## Privacy Policy
+
+H3LP is concerned about privacy of its users, especially when it comes to sensitive information such as medical data. Therefore H3LP guarantees that all conversations are end-to-end encrypted and let every user choose what data should be synchronised online.
+
+## How To Use
+
+1. Clone this repository and open it in any IDE. (Recommended AndroidStudio)
+2. Add your own API key for google map in values/google_maps_api.xml
+3. Add your own Firebase URL in com/github/h3lp3rs/h3lp/model/database/FireDatabase.kt
+4. If you want reactivate the Animation for the CPR activity (removed because of continous integration). To do so remove commented line in com/github/h3lp3rs/h3lp/view/mainpage/CprRateActivity.kt
```

Comment:
```suggestion
4. If you want to reactivate the Animation for the CPR activity (removed because of continuous integration). To do so remove commented lines in com/github/h3lp3rs/h3lp/view/mainpage/CprRateActivity.kt
```